src/query/write_query.rs

```diff
@@ -93,8 +93,8 @@ pub enum Type {
     Text(String),
 }
 
-impl ToString for Type {
-    fn to_string(&self) -> String {
+impl Into<String> for Type {
```

Comment:
I believe that this should be an implementation of `Display` rather than `Into<String>`, which will auto-implement `ToString`. This makes more sense since the types you are using don't implement `Into<String>` themselves (e.g. [`bool`](https://doc.rust-lang.org/std/primitive.bool.html#implementations)])